src/storm/CMakeLists.txt

```
@@ -56,6 +56,7 @@ add_dependencies(storm resources)
 target_link_libraries(storm PUBLIC ${STORM_DEP_TARGETS} ${STORM_DEP_IMP_TARGETS} ${STORM_LINK_LIBRARIES})
 list(APPEND STORM_TARGETS storm)
 set(STORM_TARGETS ${STORM_TARGETS} PARENT_SCOPE)
+target_precompile_headers(storm PRIVATE <any> <map> <vector> <optional> <iostream> <set> <fstream> <boost/container/flat_set.hpp> <boost/dynamic_bitset.hpp> <boost/range/irange.hpp>)
```

How is it decided what is added here? Is there some guideline? Maybe add a comment in the cmake file

```suggestion
target_precompile_headers(storm PRIVATE <any> <map> <vector> <optional> <iostream> <set> <fstream> <functional> <memory> <string> <type_traits> <unordered_map> <unordered_set> <boost/container/flat_set.hpp> <boost/dynamic_bitset.hpp> <boost/range/irange.hpp>)
```

---

I am fixing the bug on arch and will also make sure that I discuss how this is decided.